webviews/welcome-page/src/components/NewsSection.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, Flex } from '@axonivy/ui-components';
+import { useSyncExternalStore } from 'react';
+import { useTranslation } from 'react-i18next';
+import { FeedStore, type NewsItem } from '../util/FeedStore';
+import { useVscode } from '../util/useVscode';
+import './NewsSection.css';
+import { SectionButton } from './SectionButton';
+
+export const NewsSection = () => {
+  const { t } = useTranslation();
+  const { openUrl } = useVscode();
+
+  const feed = useSyncExternalStore(FeedStore.subscribe, FeedStore.getFeed);
+
+  const generateFeed = () => {
+    if (!feed) {
+      return <span>{t('welcomePage.noNewsToShow')}</span>;
+    }
+    return feed.items.slice(0, 3).map(entry => {
+      return <NewsEntry key={entry.title} entry={entry} />;
+    });
+  };
+
+  return (
+    <Flex direction='column' gap={4} style={{ height: '100%' }}>
+      <Flex directon='row' justifyContent='space-between'>
+        <h2>{t('News')}</h2>
+        <SectionButton onClick={() => openUrl('https://www.axonivy.com/blog')}>{t('welcomePage.showAll')}</SectionButton>
+      </Flex>
+      <div className='welcome-news-grid'>{generateFeed()}</div>
```

Comment:
IMO plain functions in JSX should be used as less as possible.
I would do this just inline

```suggestion
      <div className='welcome-news-grid'>
        {
          feed ? feed.items.slice(0, 3).map(entry => <NewsEntry key={entry.title} entry={entry} />)
          : <span>{t('welcomePage.noNewsToShow')}</span>
        }
      </div>
```